includes/admin/class-wc-rest-payments-reader-controller.php

```diff
@@ -292,4 +301,143 @@ function ( bool $served, WP_HTTP_Response $response ) : bool {
 			[ 'Content-Type' => 'text/html; charset=UTF-8' ]
 		);
 	}
+	/**
+	 * Returns HTML to preview a print receipt
+	 *
+	 * @param WP_REST_Request $request Full data about the request.
+	 * @return WP_HTTP_Response|WP_Error
+	 * @throws \RuntimeException Error collecting data.
+	 */
+	public function preview_print_receipt( WP_REST_Request $request ) {
+		return rest_ensure_response(
+			$this->receipts_service->get_receipt_markup(
+				$this->create_print_preview_receipt_settings_data( $request->get_json_params() ),
+				$this->create_print_preview_receipt_order_data(),
+				$this->create_print_preview_receipt_charge_data()
+			)
+		);
+	}
+
+	/**
+	 * Creates settings data to be used on the printed receipt preview. Defaults to Stripe data if one key is missing.
```

Comment:
I'm not sure what "Stripe data" means here.

Reply:
Thanks @anu-rock I'll try to explain it better. I am trying to say that it defaults to the stored settings :)

Reply:
Done